content/en/users/tutorials/filetransfer-gridstorage/_index.md

```diff
@@ -8,4 +8,229 @@ description: >
   Use EGI File Transfer to handle data in grid storage
 ---
 
-TBD
+<!-- Adjust page weight to insert this tutorial in the right place, then remove this -->
+
+## Overview
+
+This tutorial describes how to perform a transfer usin FTS transfers services
+and WebFTS.
+
+## Prerequisites
+
+As first step please make sure that you have installed the FTS client as
+described in [Data Transfer](../../data-transfer/), and in particular
+[Clients](../../data-transfer/clients/) for the command line FTS and to have
+yourcertificates installed in your browser to use WebFTS
+[WebFTS](../../data-transfer/webfts/) browser based client.
+
+To access services and resources in the
+[EGI Federated Cloud](../../getting-started), you will need:
+
+- An EGI ID (account), you can [sign up here](../../check-in/signup)
+- Enrollment into a [Virtual Organisation](../../check-in/vos) (VO) that has
+  access to the services and resources you need
+
+## FTS
+
+### Step 1 Configuration check
+
+To verify that everything is configured properly you can check with the
+following command pointing to the cerficates directly:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ fts-rest-whoami --key .globus/userkey.pem --cert .globus/usercert.pem -s https://fts3-public.cern.ch:8446/
+User DN: /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
```

Comment:
Maybe we should try to always use some fake/generic user like a Jane Doe instead of using our real identities.
@enolfc what do you think?

Reply:
I always changed or (cut part of) the resource ids in the example for the docu pages I wrote